test/delete.test.js

```diff
@@ -40,6 +40,29 @@ describe('record deletion', function() {
             });
     });
 
+    it('can throw an error if a single record delete fails', function(done) {
+        testExpressApp.set('handler override', function(req, res) {
+            res.status(402).json({
+                error: {message: 'foo bar'},
+            });
+        });
+
+        return airtable
+            .base('app123')
+            .table('Table')
+            .destroy('rec123')
+            .then(
+                function() {
+                    throw new Error('Promise unexpectly fufilled.');
+                },
+                function(err) {
+                    expect(err.statusCode).toBe(402);
+                    expect(err.message).toBe('foo bar');
+                    done();
```

Comment:
The `done` callback can be omitted when tests return a Promise (here and below)

Reply:
Done!

Reply:
Digging the pun